core/lib/spree/event/adapters/event_bus.rb

```diff
@@ -34,9 +34,10 @@ def initialize
 
         # @api private
         def fire(event_name, opts = {})
-          event = Event.new(payload: opts)
-          listeners_for_event(event_name).each do |listener|
-            listener.call(event)
+          Event.new(payload: opts).tap do |event|
```

Comment:
should we specify the change in this method return value? Returning the event instead of the listeners seems a relevant change and improvement to me, maybe we should be very explicit about that.

Reply:
In fact, I thought I had done it :upside_down_face: Added, and thanks for pointing it out!